### PR TITLE
Use modules instead of alias for added flexibility

### PR DESCRIPTION
**Pull request by zephraph**
See #20.

### CODE REVIEW

**Comment by egoist** [comment]
Maybe change the option name? Use something like `resolveRoot` instead of `aliasDefault` for this.

**Comment by zephraph** [comment]
That's doable. Would you like to keep the alias configuration as is then?

**Comment by egoist** [comment]
Yep, I think sometimes `alias` is still useful. 😅

**Comment by zephraph** [comment]
Should it be `resolveRoot` or would you rather have it more transparent that it's the modules configuration? Like `resolveModules` where it defaults to an empty array or something. I'm open to either.

File: lib/update-config.js [hunk 1]
```diff
@@ -234,11 +234,22 @@ module.exports = co.wrap(function * (webpackConfig, options) {
...
       ? options.aliasDefault
       : 'src'
-    webpackConfig.resolve.modules.push(_.cwd(srcPath))
+    webpackConfig.resolve.alias = {
+      src: _.cwd(srcPath),
+      components: _.cwd(srcPath, 'components'),
+      store: _.cwd(srcPath, 'vuex', 'store.js'),
+      css: _.cwd(srcPath, 'css'),
+      views: _.cwd(srcPath, 'views')
+    }
```
Comment (egoist) [inline lib/update-config.js]: Maybe we no longer need `aliasDefault` 😃

Comment (zephraph) [inline lib/update-config.js]: Wouldn't that technically be considered a breaking change? I mean, I'm fine with that I suppose, but you'd need to release a major version update.

**Comment by egoist** [comment]
I think `resolveRoot` is fine for now

File: lib/update-config.js [hunk 2]
```diff
@@ -245,6 +245,11 @@ module.exports = co.wrap(function * (webpackConfig, options) {
     webpackConfig.resolve.alias = options.alias
   }
 
+  // add resolveRoot
+  if (options.resolveRoot) {
+    webpackConfig.resolve.modules.push(_.cwd(options.resolveRoot))
```
Comment (egoist) [inline lib/update-config.js]: resolveRoot defaults to `src`

Comment (zephraph) [inline lib/update-config.js]: You want it to always be on by default?

Comment (zephraph) [inline lib/update-config.js]: You have write permission, feel free to update it how you see fit.

**Pull request merged**